pkg/autoscaler/autoscaler/util.go

```diff
@@ -57,6 +57,25 @@ func checkStsAutoScalingPrerequisites(set *appsv1.StatefulSet) bool {
 	return true
 }
 
+func checkStsReadyAutoScalingTimestamp(tac *v1alpha1.TidbClusterAutoScaler, threshold int32) (bool, error) {
+	if tac.Annotations == nil {
```

Comment:
I think this function should add `v1alpha1.MemberType` as param thus the tikv could also use this function.

Reply:
And also please add the note to explain the function usage and add an unit-test to cover it.